src/TuDa.CIMS.Web/Services/IStudentApi.cs

```diff
@@ -89,7 +89,7 @@ UpdateStudentDto updateModel
     /// </returns>
     /// <remarks>This will be generated by <see cref="Refit"/>.</remarks>
     [Patch("/{workingGroupId}/students/{id}")]
-    protected Task<IApiResponse<WorkingGroup>> UpdateAsyncInternal(
+    protected Task<IApiResponse<Updated>> UpdateAsyncInternal(
```

Comment:
🔧 The internal ones need to not accept any body
```suggestion
    protected Task<IApiResponse> UpdateAsyncInternal(
```